src/System.Reflection.DispatchProxy/src/System.Reflection.DispatchProxy.csproj

```diff
@@ -22,6 +22,16 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uapaot-Windows_NT-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uapaot-Windows_NT-Release|AnyCPU'" />
+  <!-- Setting PackageDestination is required for validation to succeed since we are harvesting an aot netcore50 asset which we don't want to apply
```

Comment:
Actually I think you need the uapaot configuration of this guy.  I think the right thing to do is add a configuration for uapaot10.0.16299

Reply:
will add